### PR TITLE
Change ping doc string

### PR DESCRIPTION
**Pull request by jeevanshah07**
Change the ping doc string into 'laymans terms'

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": "/home/adam/.local/share/virtualenvs/Malcolm-next-EDaiGMpS/bin/python3",
```
Comment (TheOtherUnknown) [inline .vscode/settings.json:2]: Mayyyybe don't change my config files. This may need to be in gitignore...

File: src/cogs/utils.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ async def info(self, ctx):
 
     @commands.command()
     async def ping(self, ctx):
-        """Displays latency between client and bot"""
+        """The time it takes for disords servers to respond to Malcolm"""
```
Comment (TheOtherUnknown) [inline src/cogs/utils.py]:
```suggestion
        """The time it takes for Discord's servers to respond to Malcolm"""
```

**Review by TheOtherUnknown — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (jeevanshah07) [reply .vscode/settings.json:2]: It just changes automatically when I run the code. I think putting it the .gitignore might be for the best.

Comment (TheOtherUnknown) [reply .vscode/settings.json:2]: Then you need to do that, and revert the change.

**Review by TheOtherUnknown — CHANGES_REQUESTED** [review]
Still not resolved from last review.

**Comment by jeevanshah07** [comment]
#13

**Pull request closed without merging**